modules.d/90overlay-root/overlay-mount.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh
+
+# make a read-only nfsroot writeable by using overlayfs
+# the nfsroot is already mounted to $NEWROOT
+# add the parameter rootovl to the kernel, to activate this feature
+
+. /lib/dracut-lib.sh
+
```

Comment:
I recommend `set -eu`

Reply:
Unfortunately, yes... IMO it's still better to bail out early if something fails than to continue like nothing bad happened. In the case where the mount fails, it would actually exit 1 (by chance?) because the last `mount --move` would fail and such set the script's return value. The errors until that point would be completely ignored though.

Reply:
First test using set -eu:
lib/dracut-lib.sh: line 61: DRACUT_SYSTEMD unbound variable
_newoption: unbound variable
DRACUT_SYSTEMD: unbound variable
then kernel panic
 :-(

Reply:
It may have to be done after sourcing `lib/dracut-lib.sh`...